cmd/meroxa/root/apps/deploy.go

```diff
@@ -655,7 +662,9 @@ func (d *Deploy) waitForDeployment(ctx context.Context, depUUID string) error {
 				return fmt.Errorf("failed to deploy Application %q", d.appName)
 			}
 		case <-cctx.Done():
-			return cctx.Err()
+			return fmt.Errorf(
+				"Application Deployment did not finish within %d minutes. Check `meroxa apps logs` for further information.",
```

Comment:
```suggestion
				"application deployment did not finish within %d minutes. Check `meroxa apps logs` for further information",
```

Reply:
@janelletavares I hope this resolves the linting error, but let me know what you think!